src/main/java/org/aion/wallet/ui/components/partials/UnlockAccountDialog.java

```diff
@@ -42,6 +43,7 @@
     @Override
     public void initialize(final URL location, final ResourceBundle resources) {
         registerEventBusConsumer();
+        Platform.runLater(() -> unlockPassword.requestFocus());
```

Comment:
Thanks for this one

Reply:
:+1:

Reply:
This is my single most satisfying feature